layers/+tools/shell/packages.el

```diff
@@ -165,6 +165,9 @@
       ;; multi-term commands to create terminals and move through them.
       (spacemacs/set-leader-keys-for-major-mode 'term-mode
         "c" 'multi-term
+        "C" 'term-char-mode
+        "l" 'term-line-mode
+        "N" 'multi-term-prev
         "p" 'multi-term-prev
         "n" 'multi-term-next))))
```

Comment:
Sort the keys alphabetically like in the readme: with `n` before `N`, which results in the three last lines looking like this:
```
        "n" 'multi-term-next
        "N" 'multi-term-prev
        "p" 'multi-term-prev))))
```